R/updateNullModOutcome.R

```diff
@@ -1,47 +1,50 @@
-
-### takes an existing linear model (family has to be "gaussian"!), rank-normalize residuals and scale, and re-fit null model. If group.idx are provided, optional scale by group.idx.
-
+## will update this file name to nullModelInvNorm.R
+## takes an existing null model (family has to be "gaussian"!), rank-normalize residuals and scale, and re-fit null model. 
+## If group.idx are provided, optional scale by group.idx.
 ## the names of items in the list group.idx have to match the names of the corresponding variance components!
 
```

Comment:
I moved the `updateNullModOutcome` code into `nullModelInvNorm` (which is exported). `nullModelInvNorm` did almost nothing outside of what `updateNullModOutcome` does, so I didn't see a good reason to keep them separate